packages/sophtron-adapter/src/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { AdapterMap } from "@repo/utils";
+
+import { SophtronAdapter } from "./adapter";
+import { createSophtronVC } from "./createVc";
+import type { AdapterDependencies } from "./models";
+import { SOPHTRON_ADAPTER_NAME } from "./constants";
+
+export const getSophtronAdapterMapObject = (
+  dependencies: AdapterDependencies,
+) => {
+  return {
+    sophtron: {
+      testInstitutionAdapterName: SOPHTRON_ADAPTER_NAME,
```

Comment:
This variable is if you need to route to something besides the name of the provider. No need to have it here.